torch/__init__.py

```diff
@@ -1584,7 +1578,12 @@ def compile(model: Optional[Callable] = None, *,
     Args:
        model (Callable): Module/function to optimize
        fullgraph (bool): Whether it is ok to break model into several subgraphs
-       dynamic (bool): Use dynamic shape tracing
+       dynamic (bool): Use dynamic shape tracing.  When this is True, we will up-front attempt
+        to generate a kernel that is as dynamic as possible to avoid recompilations when
+        sizes change.  This may not always work as some operations/optimizations will
```

Comment:
hmm, maybe I miss where the docs were added, but this does not explain about cuda graphs / their requirements / compilation cache / how to change its size. how to track if cuda graphs were used and when recompilations happen

Reply:
updated again, PTAL